tonic/models/vic/netcdf2vic.py

```diff
@@ -70,11 +74,11 @@ def _run(args):
             for j, key in enumerate(var_keys):
                 data[:, j] = d[key][:, y, x]
 
-            if output['Binary']:
-                write_binary(data * binary_mult, point, binary_type,
-                             out_prefix, paths['BinaryoutPath'], append)
-            if output['ASCII']:
-                write_ascii(data, point, out_prefix, paths['ASCIIoutPath'],
+            #if output['Binary']:
+            #   write_binary(data * binary_mult, point, binary_type,
+            #               out_prefix, paths['BinaryoutPath'], append)
+            #if output['ASCII']:
+            write_ascii(data, point, out_prefix, paths['ASCIIoutPath'],
```

Comment:
Not sure why this was changed but we should make sure the example configuration file works with both Binary and ASCII output.